test/Emitter/EmitterStackTest.php

```diff
@@ -110,16 +110,31 @@ public function testEmitLoopsThroughEmittersUntilOneReturnsNonFalseValue()
         $second->emit(Argument::type(ResponseInterface::class))
             ->willReturn(null);
 
-        $third = $this->prophesize(EmitterInterface::class);
-        $third->emit(Argument::type(ResponseInterface::class))
-            ->willReturn(false);
```

Comment:
The above was in the test for a specific reason: to ensure that the stack does not execute the third emitter once the second returns a non-false value. Perhaps the better solution would be to have it raise an exception:

```php
->will(function () {
    $this->fail('Should not reach third emitter');
})
```

I can make that change on merge.